content/documentation/distributed-tracing/index.adoc

```diff
@@ -20,51 +20,40 @@ toc::[]
 
 Kiali shows tracing of your applications. We currently support Jaeger version 1.9 and newer (link:https://www.jaegertracing.io/docs/1.12/frontend-ui/#embedded-mode[Read docs, window="_blank"]).
 
-icon:bullhorn[size=2x]{nbsp} To embed Jaeger, you must have the same protocol in Kiali and Tracing service.
+icon:bullhorn[size=2x]{nbsp} To have the tracing integration, you must set the configuration in external services / tracing.
 
-== Kiali and Tracing Service have the same protocol
+== Tracing url set
 
-When Kiali and Tracing service have the same protocol, we'll have the embedded feature in the website.
+When Tracing url is set, Kiali show a link to the Distributed Tracing in the menu.
 
-In the Distributed Tracing link in the menu we can see a full view for search the traces of our services.
+image::menu_external_link.png[Distributed Tracing View]
 
-image::distributed_tracing_same_protocol.png[Distributed Tracing View]
+== Tracing in_cluster_url set
 
+In each service we'll have a new tab to display the traces for the service in a scatter plot. (If we have error traces this tab will display them)
 
-In each service we'll have a new tab to display the traces for the service. (If we have error traces this tab will display them)
+image::service_traces_tab.png[Service View]
 
-image::service_view.png[Service View]
 
-We can interact with this embedded feature like the Jaeger UI
+We can interact with the tracing tool with the external links, for example "View Trace in Tracing" goes to the tracing tool trace view.
 
-image::trace_view.png[Trace View]
 
-=== Tracing Toolbar
-
-Kiali have a toolbar to control this feature to facilitate the user experience with the tracing service. When we go to the tracing view we'll see a small form as the image below where we can select the service and the lookback(Traces from last minutes or hours).
-
-image::small_form.png[Smal Form Tracing View]
+image::trace_view_jaeger.png[Trace View]
 
+=== Tracing Toolbar
 
-The service selector has the list of services of the selected namespaces organized by namespace.
-
-image::service.png[Service Selector Tracing View]
-
+Kiali have a toolbar to control this feature to facilitate the user experience with the tracing service. When we go to the tracing view we'll a form as the image below where we can select the traces interval, results limit, status code, errors, adjust time (expand results on time), last Xm traffic (Traces from last minutes)  and refresh time.
```

Comment:
Kiali have -> Kiali has
(Traces from last minutes)  and refresh time -> extra space between ) and next word.